deploy/crds/openebs.io_blockdevices_crd.yaml

```diff
@@ -238,3 +259,4 @@ spec:
         type: object
     served: true
     storage: true
+    subresources: {}
```

Comment:
Why is this field added here? The subresource field is not required. It can be removed.